source/data/home.yaml

```diff
@@ -1,111 +1,107 @@
 - id: home
-  title: 'Documentation Topics'
+  title: "Pantheon Docs"
   call-to-action:
-    title: 'Getting Started Guide'
-    sub-title: 'New to the Pantheon platform? Check out our step-by-step guide to learn all the basics.'
-    url: '/guides/getstarted/'
-    type: 'guide'
+    subtitle: "New to the Pantheon platform? Check out our step-by-step guide to learn all the basics."
+    url: "/guides/getstarted/"
+    linkText: Getting started guide
+    imageSrc: "https://cdn.bfldr.com/MEM5087K/at/wprcks7sx3wgqghb3q84p5/WEB-2x1-ProductDetailFrontEndSites-Hero_v2.psd?format=png&crop=1187%2C801%2Cx11%2Cy198"
```

Comment:
What is this domain name? Can we instead put this image inside this repo?

Reply:
@stevector — This is a Brandfolder CDN link. Brandfolder is our new DAM that is replacing Bynder. One of the reasons we chose it was for CDN capability. We hope to eventually host these decorative-type images from there for all sites. 

Are you seeing performance issues?